integration-tests/rpc-nodes.spec.ts

```diff
@@ -31,12 +32,292 @@ CONFIGS().forEach(
+                        protocol: `${protocol}`,
+                        signature:
+                            'edsigtcagjMz6xtr45ummgsxgj7V6tQGRMerspzSVLJuE2bmBv6dGffCXYqokNgymY7uY7c97kpFrzMr5dhjqwKGsUb6kSP3B97'
+                    };
+
+                    await rpcClient.preapplyOperations([operation]);
+                } catch (ex: any) {
+                    expect(ex.message).toMatch('contract.counter_in_the_past');
+                }
+                done();
+            });
+
+            it('erify that rpcClient.getEntrypoints for known contract returns list of entrypoints of the contract', async (done) => {
+                const entrypoints = await rpcClient.getEntrypoints(knownContract);
+                expect(entrypoints).toBeDefined();
+                done();
+            });
+
+            it('Verify that rpcClient.getChainId returns chain ID', async (done) => {
+                const chainId = await rpcClient.getChainId();
+                expect(chainId).toBeDefined();
+                done();
+            });
+
+            it('Verify that rpcClient.runOperation for an operation runs an operation without signature checks', async (done) => {
+                try {
+                    const operation: any = {
+                        chain_id: 'NetXjD3HPJJjmcd',
+                        operation: {
+                            branch: 'BLzyjjHKEKMULtvkpSHxuZxx6ei6fpntH2BTkYZiLgs8zLVstvX',
+                            contents: [
+                                {
+                                    kind: 'origination',
+                                    counter: '1',
+                                    source: await Tezos.signer.publicKeyHash(),
+                                    fee: '10000',
+                                    gas_limit: '10',
+                                    storage_limit: '10',
+                                    balance: '0',
+                                    script: {
+                                        code: tokenBigmapCode,
+                                        storage: tokenBigmapStorage
+                                    }
+                                }
+                            ],
+                            signature:
+                                'edsigtcagjMz6xtr45ummgsxgj7V6tQGRMerspzSVLJuE2bmBv6dGffCXYqokNgymY7uY7c97kpFrzMr5dhjqwKGsUb6kSP3B97'
+                        }
+                    };
+
+                    await rpcClient.runOperation(operation);
+                } catch (ex: any) {
+                    expect(ex.message).toMatch('contract.counter_in_the_past');
+                }
+                done();
+            });
+
+            it('getSuccessorPeriod', async (done) => {
+                const successorPeriod = await rpcClient.getSuccessorPeriod();
+                expect(successorPeriod).toBeDefined();
+                done();
+            });
         });
 
         it('List the ancestors of the head block', async (done) => {
```

Comment:
This test and the following ones seem to be duplicated.

Reply:
removed duplicate tets

Reply:
@michaelkernaghan It seems the duplicates are still there